### ISSUE TITLE
J 3.3 result not loading properly 

### ISSUE THREAD

**Issue by sajupm** [todordev/ITPGoogleSearch#8]
I use j 3.3 with itpgs 2.5 Lite ( Stable ). The search results are displayed in a pop up screen. Not like the demo site. If detailed information needed let me know

**Comment by sajupm** [comment]
I also like to request a future. Can you add a option to show the google ads @ top or right side of result. Now it's showing in top.

**Comment by todordev** [comment]
Hi,
Thank you for the post!

The extension displays ads. Do you mean somehting like that?
![google_search_ie](https://cloud.githubusercontent.com/assets/1272206/4016400/387a6aee-2a34-11e4-950d-90c48b45a38a.png)

Could you give me an example, please? You can send me a screenshot with notes.

Thanks,
Todor

**Comment by sajupm** [comment]
Yes but I ask for a option to select where the ads to be displayed.I want to show the ads only in right side but in default it shows as in the screen shot you posted (top + right or top)

**Comment by sajupm** [comment]
<a href='http://postimage.org/' target='_blank'><img src='http://s3.postimg.org/n6qa4yklv/Screenshot_2014_08_23_14_18_07.jpg' border='0' alt="Screenshot 2014 08 23 14 18 07" /></a><br /><a target='_blank' href='http://postimage.org/'>images hosting</a><br /><br />

See in this you can see the result is displayed in a pop up frame

**Comment by sajupm** [comment]
Please view this one to 
1.The result is displayed in a pop up frame....  Not in the normal page... 
2.in this we can see the ads are displayed in both top and right side of the result but I am requesting you to add an option to select where the ads to be display (either top or right or even both as per our need)

**Comment by sajupm** [comment]
<a href='http://postimage.org/' target='_blank'><img src='http://s24.postimg.org/vmgru34yd/www_pscguide_co_20140823_3f3715b3c8d9187c9720e3f.png' border='0' alt="www pscguide co 20140823 3f3715b3c8d9187c9720e3f" /></a><br /><a target='_blank' href='http://postimage.org/'>image upload no limit</a><br /><br />

**Comment by sajupm** [comment]
I am using joomla 3.3.3 + rockettheme

**Comment by susipop** [comment]
The same for me, I don't mind that it open up in a lightbox, the problem is that it gets behind my menu, breadcumbs and other things around. I think it would be easier to fix this instead of re z-index all my elements...
Any fix around yet?

![captura de pantalla 2014-09-17 a la s 21 33 42](https://cloud.githubusercontent.com/assets/206972/4314315/6bf2ecb8-3edc-11e4-9753-44d1de4527c9.png)

using Joomla 3.3.3 + independent template that works with T3 framwork plugin.
thanks!

**Comment by susipop** [comment]
Hello all! I've found out, not without felling a little dumb, that all this are not ITP's component issues, all of this can be customized in the google custom search site, inside the setting for avery specific site you've created. Hre, I leave you with this screen shot from the diferent layouts that google offers and are passed to this marvelous component directly to your website. 
![captura de pantalla 2014-09-18 a la s 15 26 57](https://cloud.githubusercontent.com/assets/206972/4326941/33d024ac-3f74-11e4-9a7f-745aaae578b8.png)
Also, If you want to customize your results with joomla modules around, you only have to create a menu item using the ITP component, then asign the modules.
This should be marked as solved...
Thanks!
